src/lib/merkle_ledger_tests/test_converting.ml

```diff
@@ -135,6 +144,23 @@ struct
             [%test_eq: Migrated.Account.t] stored_migrated_account
               (Db_converting.convert account) ) )
 
+  let () =
+    add_test "add an account, migrate, retrieve" (fun () ->
+        with_primary ~f:(fun primary ->
+            let account = Quickcheck.random_value Account.gen in
+            let location = create_new_primary_account_exn primary account in
+            with_migrated ~f:(fun migrated ->
+                (* We don't need the actual converting ledger for this test,
+                   only the side effect of migration *)
+                let _converting =
+                  Db_converting.create_with_migration primary migrated
+                in
+                let stored_migrated_account =
```

Comment:
I'd consider extending the test to add a bunch of accounts, then check that the number of accounts is the same, and the addresses are same in both (they will be iterated by `foldi` in the same order)